ios/brave-ios/Sources/Brave/Frontend/Browser/New Tab Page/Backgrounds/NewTabPageBackgroundButtonsView.swift

```diff
@@ -142,12 +159,52 @@ class NewTabPageBackgroundButtonsView: UIView, PreferencesObserver {
         $0.centerX.equalToSuperview()
       }
     }
+
+    updatePlayButtonVisibility()
   }
 
   @objc private func tappedButton(_ sender: UIControl) {
     tappedActiveButton?(sender)
   }
 
+  @objc private func tappedVideoPlayButton() {
+    tappedPlayButton?()
+  }
+
+  @objc private func tappedVideoDuringAutoplay() {
+    tappedBackgroundDuringAutoplay?()
+  }
+
+  func videoAutoplayStarted() {
+    let tapGesture = UITapGestureRecognizer(
+      target: self,
+      action: #selector(self.tappedVideoDuringAutoplay)
+    )
+    addGestureRecognizer(tapGesture)
+    playButtonGestureRecognizer = tapGesture
+  }
+
+  func videoAutoplayFinished() {
+    shouldShowPlayButton = true
+    updatePlayButtonVisibility()
+
+    if let playButtonGestureRecognizer = playButtonGestureRecognizer {
+      removeGestureRecognizer(playButtonGestureRecognizer)
+    }
+    playButtonGestureRecognizer = nil
+  }
+
+  private func updatePlayButtonVisibility() {
+    let isLandscape = frame.width > frame.height
```

Comment:
I could be mistaken but I am sure I saw a comment from @kylehickinson to change this?